src/components/home.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import {useEffect, useState} from 'react'
+import { getAllArticles } from '../utils/api'
+import { Link } from 'react-router-dom'
+
+
+function Home() {
+    const [allArticles, setAllArticles] = useState([])
+
+    useEffect(() => {
+        getAllArticles()
+        .then(({data}) => {
+            setAllArticles(data.articles)
+        })
+    })
+
+  return (
```

Comment:
I'd pop a loading tag in here using conditional rendering, for occasions when the API response might be slow.